mobile/src/components/chat-conversation/components/message-form/message-form.tsx

```diff
@@ -29,9 +29,10 @@ const MessageForm: FC<Props> = ({ onSubmit }) => {
 
   const [messageChar, setMessageChar] = useState<string>();
   const [rowsCount, setRowsCount] = useState<number>(1);
+  const [disabledButtom, setDisabledButtom] = useState<boolean>(true);
```

Comment:
Check B7 (and spelling)
```suggestion
  const [isDisabledButton, setIsDisabledButton] = useState<boolean>(true);
```

Reply:
Oh, my bad